src/AppBundle/Validator/Constraints/Recaptcha.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Sententiaregum project.
+ *
+ * (c) Maximilian Bosch <maximilian.bosch.27@gmail.com>
+ * (c) Ben Bieler <benjaminbieler2014@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace AppBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+/**
+ * Validation constraint for captchas.
+ *
+ * @author Benjamin Bieler <benjaminbieler2014@gmail.com>
+ *
+ * @Annotation
+ */
+class Recaptcha extends Constraint
+{
+    /**
+     * Hash which the user creates.
+     *
+     * @var RecaptchaHash (required!!)
```

Comment:
not necessary to add here a note that it's required. If it doesn't exist, an exception will be thrown